tests/test_transformations.py

```diff
@@ -0,0 +1,177 @@
+from pathlib import Path
+import math
+import random
+import os
+
+import pytest
+import h5py
+import numpy as np
+import cv2
+import imageio
+
+import segmentation_labeling_app.transforms.transformations as transformations
+
+
+@pytest.fixture()
+def video_fixture():
+    current_path = (Path(__file__)).parent
+    video_path = current_path / 'resources' / 'test_video.h5'
+    yield video_path
+
+
+def test_video_random_downsampling(video_fixture):
```

Comment:
Good coverage of the different types. I think it would be a lot less verbose if you parametrized the tests and included the downsampling type argument in the parametrization.

Reply:
I agree that it would be easier to read if parametrized, the seed is now controlled so that won't be an issue